browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -291,13 +282,75 @@ class MarkdownNoteDetail extends React.Component {
                 </g>
               </svg>
             </button>
-            <button styleName='control-fullScreenButton'
-              onMouseDown={(e) => this.handleFullScreenButton(e)}
-            >
-              <i className='fa fa-arrows-alt' styleName='fullScreen-button' />
-            </button>
+
+          // TODO: must refactor
+          const menuBar = location.pathname === '/trashed'
```

Comment:
Obviously, this code is dirty. So I assume I have to make some new components for it.